### ISSUE TITLE
Home page query output isn't what I expected

### ISSUE THREAD

**Issue by bobbingwide** [bobbingwide/tt4ai#2]
Having added the `aiprompt` and `airesponse` styles I wanted to try them on wp-secrets.
But when I uploaded the new child theme and made it active I lost the customization I'd applied for Twenty Twenty-Four (TT4)
So I reverted to TT4 and copied the home template source into tt4ai.
In my local machine this template worked in the site editor
but it produced the incorrect output on the front end.
### Expected output
For the Plugins by title
- oik-plugins only
- 3 per page

### Actual output
- posts 
- 4 per page - since it includes the post marked "Stick to the top of the blog"
![image](https://github.com/bobbingwide/tt4ai/assets/2474435/2ecc7608-8e72-480a-8168-7858839fb81a)

**Comment by bobbingwide** [comment]
Exactly the same query, when run in a post does produce the correct output in both the block editor and the front end.

![image](https://github.com/bobbingwide/tt4ai/assets/2474435/2460e14f-85a3-4e6c-a5d7-4febd0aeb845)

**Comment by bobbingwide** [comment]
The problem occurs both for _Your homepage displays Your Latest posts_ and _Your home page displays Homepage_
but it works when you view the Blog page.

**Comment by bobbingwide** [comment]
Also just got a fatal error visiting https://s.b/wordpress/wp-admin/edit.php?post_type=wp_template_part
after deactivating Gutenberg. 

```
Fatal error: Uncaught ArgumentCountError: 3 arguments are required, 2 given 
in C:\apache\htdocs\wordpress\wp-includes\link-template.php:1580 
Stack trace: 
#0 C:\apache\htdocs\wordpress\wp-includes\link-template.php(1580): sprintf('site-editor.php...', 21375) 
#1 C:\apache\htdocs\wordpress\wp-admin\includes\class-wp-posts-list-table.php(1515): get_delete_post_link(Object(WP_Post)) 
#2 C:\apache\htdocs\wordpress\wp-admin\includes\class-wp-posts-list-table.php(1073): WP_Posts_List_Table->handle_row_actions(Object(WP_Post), 'title', 'title') 
#3 C:\apache\htdocs\wordpress\wp-admin\includes\class-wp-list-table.php(1790): WP_Posts_List_Table->_column_title(Object(WP_Post), 'title column-ti...', 'data-colname="T...', 'title') 
#4 C:\apache\htdocs\wordpress\wp-admin\includes\class-wp-posts-list-table.php(1432): WP_List_Table->single_row_columns(Object(WP_Post)) 
#5 C:\apache\htdocs\wordpress\wp-admin\includes\class-wp-posts-list-table.php(834): WP_Posts_List_Table->single_row(Object(WP_Post), 0) 
#6 C:\apache\htdocs\wordpress\wp-admin\includes\class-wp-posts-list-table.php(810): WP_Posts_List_Table->_display_rows(Array, 0) 
#7 C:\apache\htdocs\wordpress\wp-admin\includes\class-wp-list-table.php(1709): WP_Posts_List_Table->display_rows() 
#8 C:\apache\htdocs\wordpress\wp-admin\includes\class-wp-list-table.php(1636): WP_List_Table->display_rows_or_placeholder() 
#9 C:\apache\htdocs\wordpress\wp-admin\edit.php(503): WP_List_Table->display() 
#10 {main} thrown in C:\apache\htdocs\wordpress\wp-includes\link-template.php on line 1580
```

**Comment by bobbingwide** [comment]
The code that's not working on the home page is
```
<!-- wp:group {"align":"wide","layout":{"type":"constrained"}} -->
<div class="wp-block-group alignwide"><!-- wp:query {"queryId":0,"query":{"perPage":"3","pages":0,"offset":0,"postType":"oik-plugins","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":false,"parents":[]},"align":"wide","layout":{"type":"default"}} -->
<div class="wp-block-query alignwide"><!-- wp:post-template -->
<!-- wp:separator {"backgroundColor":"contrast-3","className":"alignwide is-style-wide"} -->
<hr class="wp-block-separator has-text-color has-contrast-3-color has-alpha-channel-opacity has-contrast-3-background-color has-background alignwide is-style-wide"/>
<!-- /wp:separator -->

<!-- wp:columns {"verticalAlignment":"center","align":"wide","style":{"spacing":{"margin":{"top":"var:preset|spacing|20","bottom":"var:preset|spacing|20"}}}} -->
<div class="wp-block-columns alignwide are-vertically-aligned-center" style="margin-top:var(--wp--preset--spacing--20);margin-bottom:var(--wp--preset--spacing--20)"><!-- wp:column {"verticalAlignment":"center","width":"27.45%"} -->
<div class="wp-block-column is-vertically-aligned-center" style="flex-basis:27.45%"><!-- wp:post-title {"isLink":true,"style":{"typography":{"lineHeight":"1.1","fontSize":"1.5rem"}}} /-->

<!-- wp:post-featured-image {"isLink":true} /--></div>
<!-- /wp:column -->

<!-- wp:column {"verticalAlignment":"center","width":"39.22%"} -->
<div class="wp-block-column is-vertically-aligned-center" style="flex-basis:39.22%"><!-- wp:oik-block/fields /--></div>
<!-- /wp:column -->

<!-- wp:column {"width":"33.33%"} -->
<div class="wp-block-column" style="flex-basis:33.33%"><!-- wp:post-terms {"term":"category"} /--></div>
<!-- /wp:column --></div>
<!-- /wp:columns -->
<!-- /wp:post-template -->

<!-- wp:spacer {"height":"var:preset|spacing|30"} -->
<div style="height:var(--wp--preset--spacing--30)" aria-hidden="true" class="wp-block-spacer"></div>
<!-- /wp:spacer -->

<!-- wp:query-pagination {"paginationArrow":"arrow","layout":{"type":"flex","justifyContent":"space-between"}} -->
<!-- wp:query-pagination-previous /-->

<!-- wp:query-pagination-numbers /-->

<!-- wp:query-pagination-next /-->
<!-- /wp:query-pagination -->

<!-- wp:query-no-results -->
<!-- wp:paragraph -->
<p>No posts were found.</p>
<!-- /wp:paragraph -->
<!-- /wp:query-no-results --></div>
<!-- /wp:query --></div>
<!-- /wp:group -->
```

**Comment by bobbingwide** [comment]
I tried in a different environment ( `s.b/wp55/tt4` ) and was able to confirm that I didn't get the problem with just a few plugins;
oik, oik-fields, oik-types and oik-bwtrace.
That was once I'd resolved a few silly mistakes in registering the CPT.

**Comment by bobbingwide** [comment]
I didn't find out the details of the problem but it must have been something to do with the definition of the `oik-plugins` CPT in `s.b/wordpress`.  Once I'd deactivated all the plugins and reactivated enough to get it to work it did start working as expected

**Comment by bobbingwide** [comment]
Implemented in v0.1.0
